### PR TITLE
Completed PostgreSQL connection and queries

### PR DESCRIPTION
**Pull request by priscilagonzalez**

### CODE REVIEW

File: database/model/exp.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    text: 'SELECT * FROM experiences WHERE experienceId = $1',
+    values: [expId],
+  };
+
+  const timer = hrtime();
+  client.query(selectQuery, (err, res) => {
+    if (err) {
+      callback(err);
+    } else {
+      console.log(`Execution time: ${hrtime(timer, 's')}`);
```
Comment (Kraig5) [inline database/model/exp.js]: Remember to comment out logs like this (that will slow down your system) when testing.

File: database/model/exp.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      callback(null, res.rows);
+    }
+  });
+};
+
+const createExp = (newExp, callback) => {
+  const columns = 'title, city, state, country, category, activity, averageRating, numberOfReviews, duration, groupSize,includes, cuisine, hostedLanguages, costPerPerson, imageUrls,videoUrl';
+  const params = '$1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16';
```
Comment (Kraig5) [inline database/model/exp.js]: Wow, that's a lot of stuff!

File: database/index.js [hunk 3]
```diff
@@ -1,65 +1,17 @@
-const mongoose = require('mongoose');
-mongoose.Promise = require('bluebird');
-const Experience = require('./experience.js');
+require('dotenv').config();
+const { Client } = require('pg');
 
-mongoose.set('useCreateIndex', true);
+const user = process.env.PGUSER;
```
Comment (Kraig5) [inline database/index.js]: Good stuff -- this will make deployment easier.

File: database/model/exp.js (hunk 1 above)
Comment (priscilagonzalez) [reply database/model/exp.js]: Thank you!

**Pull request merged**